### PR TITLE
Feature: Make Phenomodels subpanels collapsible (#2338)

### PR DESCRIPTION
**Pull request by moedarrah**
This PR adds functionality to make Phenomodels subpanels collapsible

**How to test**:
1. Download branch feature/2338
2. Go to the tab Phenomodels
3. Click on a panel

**Expected outcome**:
- [x] All panels should be clickable and collapse/expand when clicked

**Review:**
- [x] code approved by CR
- [x] tests executed by JH & MOD

### CODE REVIEW

**Comment by northwestwitch** [comment]
I can review this one!

File: scout/server/blueprints/institutes/templates/overview/phenomodel.html [hunk 1]
```diff
@@ -172,9 +172,12 @@ <h6 class="text-muted ml-3 mt-3">Created: {{phenomodel.created.strftime('%Y-%m-%
     {% for subpanel_id, subpanel in phenomodel.subpanels.items()%}
       <div class="card">
         <div class="card-header">
-          <strong>{{subpanel.title}}</strong>&nbsp;&nbsp;<text class="text-muted">{{subpanel.subtitle}}</text>
+          <a href="#" onclick="toggleDiv('{{subpanel_id}}')">
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/templates/overview/phenomodel.html:175]: Panel toggle works. I noticed that all subpanels are expanded when you go to a panel page. Could we have it the other way around instead, with all subpanels hidden and shown only if you click on them?

**Review by northwestwitch — APPROVED** [review]
A small suggestion for improvement, but works already! 👍

File: scout/server/blueprints/institutes/templates/overview/phenomodel.html (hunk 1 above)
Comment (moedarrah) [reply scout/server/blueprints/institutes/templates/overview/phenomodel.html:175]: We fixed it

**Comment by northwestwitch** [comment]
Looking good: 💯 
image

Feel free to merge when you are ready!

**Pull request merged**